templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>There should be a GIF showing how to use this, to pair with extremely minimal instructions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Related to issue #79.</p>

Reply:
<p>Are we looking for a GIF per step (like the slides) or one GIF demonstrating a simplified walkthrough of the entire process?</p>

Reply:
<p>It looks to me like only bullet 1 needs a GIF. Up to you though, play around and see what is going to be easiest for a new user to understand who's never used the GUI before</p>